src/SingleArticle.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState,useEffect } from "react";
+import { useParams } from "react-router-dom";
+import getArticleById from "../api";
+
+
+function SingleArticle(){
+
+    const[article,setArticle]=useState({});
```

Comment:
Please include a state for loading and display a message to the user while the articles are loading.